app/src/main/java/com/example/sharingang/EditItemFragment.kt

```diff
@@ -15,8 +20,19 @@ import com.example.sharingang.items.ItemsViewModel
 class EditItemFragment : Fragment() {
 
     private val viewModel: ItemsViewModel by activityViewModels()
+
+    private lateinit var contextActivity: MainActivity

```

Comment:
I did that because the functions that I used were in the MainActivity. I don't know how to put these functions elsewhere than in the MainActivity, because most of the functions I call need to have an activity.